types.go

```diff
@@ -41,45 +41,67 @@ type digSentinel interface {
 	digSentinel()
 }
 
-// In is an embeddable object that signals to dig that the struct
-// should be treated differently. Instead of itself becoming an object
-// in the graph, memebers of the struct are inserted into the graph.
+// In may be embedded into structs to request dig to treat them as special
+// parameter structs. When a constructor accepts such a struct, instead of the
+// struct becoming a dependency for that constructor, all its fields become
+// dependencies instead. See the section on Parameter Objects above for more
+// information.
```

Comment:
Let's not presume that everyone's looking at the docs in a browser - can we say, "See the section on Parameter Objects in the package-level documentation for more details"?